src/content/app/tools/blast/state/blast-results/blastResultsSlice.ts

```diff
@@ -72,7 +72,7 @@ export type BlastSubmission = {
   id: string;
   submittedData: {
     species: Species[];
-    sequences: { id: number; value: string }[]; // TODO: consider whether to have strings or parsed sequences
+    sequences: { id: number; header?: string; value: string }[]; // TODO: consider whether to have strings or parsed sequences
     preset: string;
     submissionName: string;
     parameters: BlastSubmissionParameters;
```

Comment:
Shouldn't this be `BlastSubmissionPayload` type?

Reply:
we already have an export type `BlastSubmissionPayload` in blastApiSlice.ts  and in this one there is the extra part for the result which is related to a submission in blast hence just `BlastSubmission`. Anyway i didnt choose this name in this PR and it feels like it is the right name for it. Happy to discuss this.

Reply:
> Shouldn't this be BlastSubmissionPayload type?

A submission payload included, apart from BLAST parameters, a list of selected species, and a list of query sequences. The `BlastSubmissionParameters` type is only about the parameters (the ones that you select using the dropdowns in the BLAST submission form).